### DISCUSSION TITLE
[gamemode]: 未能找到依赖 minecraft_data_api

### DISCUSSION THREAD

**Discussion by LIPiston** [AnzhiZhang/MCDReforgedPlugins#112] [Q&A] [answered]
## Describe
如标题
## Details
如标题

## Expected behavior
求解决方案

## Screenshots
![FI_CW2W2G7@J3YQVDWBYZWX](https://user-images.githubusercontent.com/75552696/130325654-f70d487a-fc7a-44dd-9cce-afbbb4bde9d0.png)<br/>
![}}){ _9ZMV_QSB@UNEX8LSL](https://user-images.githubusercontent.com/75552696/130325662-14ef017c-bb29-442f-ab85-a06a95519138.png)

## Additional context
无

**Comment by AnzhiZhang** [comment, marked as answer]
你没装 hjson，这是jsonDataAPI的前置python包
![image](https://user-images.githubusercontent.com/37402126/130328277-881e9aad-9f96-46c6-9e58-10741de7a4c4.png)
